docs/reference/messenger.rst

```diff
@@ -0,0 +1,48 @@
+Messenger
+=========
+
+The bundle provides a specific thumbnail class to generate thumbnails through an asynchronous task. So there no processing
+time for the user after uploading a file.
+
+It is recommended to read about `Symfony Messenger`_ if it is your first time using it.
+
+First, you need to install Symfony Messenger:
+
+.. code-block:: bash
+
+    composer require symfony/messenger
+
+After installing the dependency, you need to enable the integration and change the thumbnail configuration
+for each provider:
+
+.. code-block:: yaml
+
+    # config/packages/sonata_media.yaml
+
+    sonata_media:
+        messenger:
+            enabled: true
+        providers:
+            image:
+                thumbnail:  sonata.media.thumbnail.consumer.messenger
```

Comment:
This is correct? I did not find the service id outside the docs.

Reply:
woops, no, the .consumer needs to be removed.

Reply:
Fixed.